src/main.cpp

```diff
@@ -836,7 +836,13 @@ write_file(const char *path, const std::string_view &contents) {
  */
 std::uint16_t
 map_port(int port) {
-  // TODO: Ensure port is in the range of 21-65535
+  // Ensure port is in the range of 1024-65535
+  if (port < 1024 || port > 65535) {
+    BOOST_LOG(warning) << "Port out of range: "sv << port;
```

Comment:
While ports < 1024 are root-only, these are still valid ports. I think in this specific case making the warning is enough

Reply:
This condition should actually never happen, as I modified the range for config port 1029-65514 (to account for the difference from the config port (-5, +21)